### ISSUE TITLE
error: linking with `cc` failed: exit code: 1

### ISSUE THREAD

**Issue by Eligioo** [nimiq/core-rs#47]
## New issue checklist

- [x] I have read all of the [`README`](https://github.com/nimiq-network/core/blob/master/README.md) 
- [x] I have searched [existing issues](https://github.com/nimiq-network/core/issues?q=is%3Aissue+sort%3Acreated-desc) and **this is not a duplicate**.

Tried to build `core-rs` from source by cloning from Github. I switched to nightly build with `rustup default nightly`. As soon as I started building with `cargo +nightly build` the compiling stopped with the following error:
```error: linking with `cc` failed: exit code: 1```

The README of `core-rs` states that besides rust nightly you need gcc, pkg-config and libssl-dev, which I installed through `apt`.
# Solution
Install `gcc-multilib` as well and it started compiling with no issues.

**Comment by Eligioo** [comment]
Weird though, after I `apt remove gcc-mutlilib`, it's still able to compile now. Guess `gcc` linker wasn't found and after installing `gcc-multilib` it was somehow.

**Comment by sisou** [comment]
Did your install of `gcc-multilib` install other packages as dependencies?

**Comment by Eligioo** [comment]
@sisou It is a fresh Hetzner server with nodejs installed already. I added `rustup` and switched to nightly. Then `apt install gcc pkg-config libssl-dev`
After installation I tried to compile the repo which failed with the error stated above. Then I did an `apt install gcc-multilib` and was able to compile the project. When it was done I removed the target folder and did `apt remove gcc-multilib` and was still able to compile.

Hope it makes sense :-)
